client/jetpack-connect/test/authorize.js

```diff
@@ -63,6 +69,24 @@ const DEFAULT_PROPS = deepFreeze( {
 	userHasUnattachedLicenses: false,
 } );
 
+const APPROVE_SSO_CLIENT_ID = 99821;
+
+jest.mock( '../persistence-utils', () => ( {
+	...jest.requireActual( '../persistence-utils' ),
+	isSsoApproved: ( clientId ) => clientId === APPROVE_SSO_CLIENT_ID,
+} ) );
+
+function renderWithRedux( ui ) {
+	return renderWithProvider( ui, {
+		reducers: {
+			ui: uiReducer,
+			documentHead: documentHeadReducer,
```

Comment:
We could also mock `<DocumentHead />` but this should do it just equally well too 👍